src/app/core/shared/community.model.ts

```diff
@@ -64,6 +63,17 @@ export class Community extends DSpaceObject implements ChildHALResource {
   @link(COMMUNITY, false)
   parentCommunity?: Observable<RemoteData<Community>>;
 
+  /**
+   * A string representing the unique handle of this Community
+   */
+  get handle(): string {
+    return this.firstMetadataValue('dc.identifier.uri');
```

Comment:
If the getter for `handle` returns `this.firstMetadataValue('dc.identifier.uri')` do we still need the setter, or the private `_handle`  property?

Reply:
The setter has been introduced because in the initial solution the old field was used as a fallback value in case the metadata was not present. But since it is not used anymore, and I don't see any reason to use that value which is not so accurate through out the entire application, I agree that its presence doesn't make much sense. I'll remove it (from both collections and communities).